\documentclass{article}
\usepackage{url}
\usepackage{breakurl}
%\VignetteIndexEntry{The DMRcate package user's guide}
%\VignetteEngine{knitr::knitr}
\begin{document}
\title{The \texttt{DMRcatedata} package user's guide}

\author{Peters TJ, Buckley MJ, Statham AL, Clark SJ, Molloy PL}
\maketitle

\section*{Package Contents}

\texttt{DMRcatedata} accompanies the \texttt{DMRcate} package, providing data for examples, probe filtering and transcript annotation.

<<libr, message=FALSE, warning=FALSE>>=
library(DMRcatedata)
data(dmrcatedata)
@

Three objects are contained in \texttt{DMRcatedata}. 
\texttt{crosshyb} is a factor listing probe IDs potentially confounded by cross-hybridisation to other parts of the genome\cite{Chen}. It is used internally by \texttt{rmSNPandCH()}.

<<crosshyb>>=
str(crosshyb)
@

\texttt{illuminaSNPs} is a data.frame containing probes that are potentially confounded by a SNP. It lists the SNPID, distance (in nucleotides) to the CpG in question, and minor allele frequency for each associated SNP

<<illuminaSNPs>>=
class(illuminaSNPs)
dim(illuminaSNPs)
head(illuminaSNPs)
@

\texttt{myBetas} is a matrix of 450K probe beta values, matching to Homo Sapiens chromosome 20, sourced from the colon and rectal adenocarcinoma section of The Cancer Genome Atlas (TCGA) Repository. It contains 38 matched tumour/normal pairs. 

<<myBetas>>=
class(myBetas)
dim(myBetas)
@


\texttt{CpGs} is a GRanges object containing simulated WGBS data, generated by WGBSSuite\cite{Rackham} v0.3 with the command \texttt{Rscript simulate\_WGBS.R 100000 0.87605280264016 0.125787302952703 0.2 0.2 18.5989386821267 18.5989386821267 3 2 0.2 0.5 0.112588288740425,0.00330228672976263 . truncated}.

<<CpGs>>=
CpGs
@

\texttt{tx.hg19}, \texttt{tx.hg38} and \texttt{tx.mm10} are GRanges objects containing complete transcript annotation generated from \url{ftp://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/Homo_sapiens.GRCh37.75.gtf.gz}, \url{ftp://ftp.ensembl.org/pub/release-81/gtf/homo_sapiens/Homo_sapiens.GRCh38.81.gtf.gz} and \url{ftp://ftp.ensembl.org/pub/release-81/gtf/mus_musculus/Mus_musculus.GRCm38.81.gtf.gz} respectively.

<<tx.hg19>>=
tx.hg19
@

\section*{Sources}
\begin{itemize}
  \item \texttt{myBetas} sourced from The Cancer Genome Atlas (TCGA) data repository, colon and rectal adenocarcinoma
  \item \texttt{illuminaSNPs} sourced from \url{http://supportres.illumina.com/documents/myillumina/88bab663-307c-444a-848e-0ed6c338ee4d/humanmethylation450_15017482_v.1.2.snpupdate.table.v3.txt}, accessed February 2014
  \item \texttt{crosshyb} sourced from \url{http://www.sickkids.ca/MS-Office-Files/Research/Weksberg Lab/48639-non-specific-probes-Illumina450k.xlsx}, accessed February 2014
  \item \texttt{tx.hg19}, \texttt{tx.hg38} and \texttt{tx.mm10} sourced from \url{ftp://ftp.ensembl.org}, accessed July 2015.
\end{itemize}

\begin{thebibliography}{9}

\bibitem{Chen}
  Chen YA, Lemire M, Choufani S, Butcher DT, Grafodatskaya D, Zanke BW, Gallinger S, Hudson TJ, Weksberg R. 
  Discovery of cross-reactive probes and polymorphic CpGs in the Illumina Infinium HumanMethylation450 microarray. 
  \emph{Epigenetics}. 2013 Jan 11;8(2).
  
\bibitem{Rackham}
  Rackham, OJL, Dellaportas P, Petretto E, Bottolo, L. (2015). WGBSSuite: Simulating Whole Genome Bisulphite Sequencing data and benchmarking differential DNA methylation analysis tools. 
  \emph{Bioinformatics} (Oxford, England), (March). \url{http://doi.org/10.1093/bioinformatics/btv114}

\end{thebibliography}

\end{document}